ports/onnx/portfile.cmake

```
@@ -0,0 +1,100 @@
+# uwp: LOAD_LIBRARY_SEARCH_DEFAULT_DIRS undefined identifier
+vcpkg_fail_port_install(ON_TARGET "uwp")
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO onnx/onnx
+    REF v1.9.0
+    SHA512 a3eecc74ce4f22524603fb86367d21c87a143ba27eef93ef4bd2e2868c2cadeb724b84df58a429286e7824adebdeba7fa059095b7ab29df8dcea8777bd7f4101
+    PATCHES
+        fix-cmakelists.patch
+        fix-pybind11.patch
+)
+
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        pybind11 BUILD_ONNX_PYTHON
+)
+
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" USE_PROTOBUF_SHARED)
+list(APPEND FEATURE_OPTIONS 
+    -DONNX_USE_PROTOBUF_SHARED_LIBS=${USE_PROTOBUF_SHARED}
+)
+if(VCPKG_TARGET_IS_WINDOWS)
+    string(COMPARE EQUAL "${VCPKG_CRT_LINKAGE}" "static" USE_STATIC_RUNTIME)
+    list(APPEND FEATURE_OPTIONS 
+        -DONNX_USE_MSVC_STATIC_RUNTIME=${USE_STATIC_RUNTIME}
+    )
+endif()
+
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+vcpkg_find_acquire_program(PYTHON3)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
```

```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```
The path should be quoted in cmake file. Please see the doc https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/cmake-guidelines.md.

---

Got it. I will check the other works either.